server/proxy/list.go

```diff
@@ -4,15 +4,19 @@
 package proxy
 
 import (
+	"context"
 	"sort"
 	"strings"
+	"time"
 
 	"github.com/mattermost/mattermost-server/v6/model"
 
 	"github.com/mattermost/mattermost-plugin-apps/apps"
 	"github.com/mattermost/mattermost-plugin-apps/server/incoming"
 )
 
+const pingAppTimeout = 1 * time.Second
```

Comment:
I think should be documented that your App respond within 1 second for this request, whether the App implements that endpoint or not